lib/ansible/modules/windows/win_wait_for.ps1

```diff
@@ -16,7 +16,7 @@ $exclude_hosts = Get-AnsibleParam -obj $params -name "exclude_hosts" -type "list
 $hostname = Get-AnsibleParam -obj $params -name "host" -type "str" -default "127.0.0.1"
 $path = Get-AnsibleParam -obj $params -name "path" -type "path"
 $port = Get-AnsibleParam -obj $params -name "port" -type "int"
-$search_regex = Get-AnsibleParam -obj $params -name "search_regex" -type "str"
+$search_regex = Get-AnsibleParam -obj $params -name "search_regex" -type "str" -aliases "regex","regexp"
```

Comment:
I'm not against adding the alias to this but not 100% sure if we should change the actual documented option name to `regex` as per the discussion as per https://github.com/ansible/ansible/issues/53402#issuecomment-509770925. What are your thoughts on this?

Reply:
I think we should change it like for win_lineinfile - so code will show the changes